feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+this file was also added in the feaure1 branch
+
+one more change ok 
```

Comment:
are you crazy ? what are you doing?

Reply:
tell me why I am crazy ? that code is correct